hasura.planx.uk/metadata/tables.yaml

```diff
@@ -347,6 +347,7 @@
           - creator_id
           - data
           - id
+          - metabase_link
```

Comment:
This indicates that only the `platformAdmin` role can access this column, when I think what we really want is for all roles used by authorised users (i.e. all but `public`) to be able to access this.

Reply:
Fixed here: 1d5c3043c60aa3f63327967d5e677ec20ad46a13